tests/integration_tests/csv_upload_tests.py

```diff
@@ -556,4 +556,4 @@ def test_import_parquet(mock_event_logger):
 
     with test_db.get_sqla_engine_with_context() as engine:
         data = engine.execute(f"SELECT * from {PARQUET_UPLOAD_TABLE}").fetchall()
-        assert data == [("max", 3), ("bob", 4), ("john", 1), ("paul", 2)]
+        assert data == [("john", 1), ("paul", 2), ("max", 3), ("bob", 4)]
```

Comment:
@hughhhh you can revert this line as it's been addressed in https://github.com/apache/superset/pull/23856.